### DISCUSSION TITLE
Load more button disappears once pressed

### DISCUSSION THREAD

**Discussion by Askancy** [livewire/livewire#5071] [1. Help]
I followed the instructions given by the user in this thread:

https://forum.laravel-livewire.com/t/implement-load-more-pagination/871/4

But once you press the button, it disappears and does not allow you to advance a page

https://user-images.githubusercontent.com/789702/175816020-3d0dd04c-3e5b-491a-ae08-ac089266f472.mp4

**Comment by joshhanley** [comment]
@Askancy can you share your component class and view so we can have a look? Thanks!
